sc/baron2016_pancreas_mouse.pkl.gz.info

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "baron2016_pancreas_mouse.pkl.gz",
+    "name": "baron2016_pancreas_mouse",
```

Comment:
The only change to this file is that I've removed the file extension from the "name" field. @lanzagar do I also need to bump the version here?

Reply:
Minor version bump sounds like the right thing.